src/lib/components/accordion/Accordion.svelte

```diff
@@ -12,11 +12,13 @@
   <slot name="title">expand</slot>
 </AccordionButton>
 
-{#if revealed}
```

Comment:
This change is required because if we do not have slot in DOM in the hidden state then adding it later via toggling changes the total width of the page content and page moves around. Hence we always would like to have accordion body in DOM and set it to be invisible and set its height and spacing to 0.